daemon/algod/api/README.md

```diff
@@ -1,5 +1,31 @@
 # Readme
 
+# V2 Endpoint
+With the V2 REST API we started using a design driven process.
+
+The API is defined using [OpenAPI v2](https://swagger.io/specification/v2/) in **algod.oas2.yml**.
+
+## Updating the V2 REST API
+Prerequisit:
+You need to install our fork of **oapi-codegen**, available here: https://github.com/algorand/oapi-codegen/
+Run `go build` in the cmd/oapi-codegen. At this point I haven't been able to get `go get` to work.
```

Comment:
"At this point I haven't been able to get `go get` to work." What do you expect the user to do with this comment? :) Consider deleting it or getting `go get` to work.

Reply:
If there is a need to have 'tools' installed and go modules are used a common pattern is to define a file like tools.go in the module root with a build tag that prevents its use in package builds.
The fetching of packages by the module system will still honor dependencies listed within it though and can be used for automatic dependencies and fetching of tools.

tools.go
----
```
// +build tools

package tools

import (
        _ "github.com/algorand/oapi-codegen"
)
```

Reply:
some of our tools are currently being installed during configure-deps.sh; would that work for this tool as well ?

Reply:
Brian was able to fix go get on Friday. Thanks for all the feedback on this, I'll get it resolved tomorrow.

Reply:
fixed.